VARIABLES.md

```diff
@@ -406,20 +414,28 @@ Default:  "{{kafka_broker_configure_additional_brokers}}"
 
 ### kafka_broker_user
 
-Only use to customize Linux User Kafka Service runs with. User must exist on host.
+Only use to customize Linux User Kafka Service runs with. Default user is cp-kafka
 
 Default:  "{{kafka_broker_default_user}}"
 
 ***
 
 ### kafka_broker_group
 
-Only use to customize Linux Group Kafka Service user belongs to. Group must exist on host.
+Only use to customize Linux Group Kafka Service user belongs to. Default user is confluent
 
 Default:  "{{kafka_broker_default_group}}"
 
 ***
 
+### kafka_broker_log_dir
+
+Only use to customize directory Kafka writes log files to. Default location is /var/log/kafka. NOTE- kafka_broker.appender_log_path is deprecated
```

Comment:
Modify this variable to customize the directory that the Kafka Broker writes log files to. Default location is /var/log/kafka. NOTE- kafka_broker.appender_log_path is deprecated